gh2/csv.py

```diff
@@ -190,19 +196,24 @@ def write_rows(filename, headers, fields, issues, date_format, include_prs,
             issue_data = issue_to_dict(fields, issue, additional_labels)
             if not skip_normalization:
                 issue_data = normalize_sequential_dates(issue_data)
+            if include_milestones:
+                milestone = issue.milestone.title if issue.milestone \
```

Comment:
hmm, I wasn't sure, but if you as a reviewer say the code would be more readable the other way then I shall change it